.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
why not add some defaults?

Reply:
Because there is nothing project specific yet. All critical files are covered by my global gitignore and I think the local one should cover local stuff that should be ignored. Please let me know how we want to handle them in the future.

Reply:
this requires everyone to have correct global gitignore files, i'm not a fan of that. there are also project specific file, like npm debug logs.